projects/aca-content/src/lib/ui/theme.scss

```diff
@@ -14,6 +14,12 @@ mat-icon {
   vertical-align: middle;
 }
 
+button {
+  #{$mat-icon} {
```

Comment:
this is too risky change, you essentially change every icon in the button in the entire app